src/main/java/com/onarandombox/MultiverseCore/listeners/MVPlayerListener.java

```diff
@@ -170,7 +170,7 @@ public void playerTeleport(PlayerTeleportEvent event) {
             return;
         }
         Player teleportee = event.getPlayer();
-        CommandSender teleporter = null;
+        CommandSender teleporter = teleportee;
```

Comment:
Any thought on why this would have resulted in an NPE? In what case does it miss assigning a value?

Reply:
Bcu we reference teleporter.getName() for some logging in some code below